### ISSUE TITLE
Error due to spree/api/payments/source_views

### ISSUE THREAD

**Issue by sechix** [solidusio/solidus_gateway#81]
Steps to reproduce
Iam getting error in orders that not load shipment and line items, I realized debugging that gets error because doesn't exist _stripe.json.jbuilder in spree/api/payments/source_views

![image](https://user-images.githubusercontent.com/24247266/39534902-ed708340-4e32-11e8-89a0-987bf969b46c.png)

This file 'sources_views' has been created 1 month ago in solidus master.

System configuration
Solidus master last version:

**Comment by octave** [comment]
Same error using `solidus_paypal_braintree` with solidus 2.6.0rc1

**Comment by sechix** [comment]
Yes, It works for me now with this workaround: I created  '_stripe.json.jbuilder'   in spree/api/payments/source_views with the same code ,

attrs = [:id, :month, :year, :cc_type, :last_digits, :name]
 if @current_user_roles.include?("admin")
   attrs += [:gateway_customer_profile_id, :gateway_payment_profile_id]
 end

json.(payment_source, *attrs)

Create the same withe braintree instead of stripe

**Comment by octave** [comment]
Thanks @sechix! Works fine now 👍
